recipe/thirdparty/providers/apple.go

```diff
@@ -87,6 +90,8 @@ func Apple(input tpmodels.ProviderInput) tpmodels.TypeProvider {
 				} else {
 					res["user"] = userInfo
 				}
+			} else if userInfo, ok := redirectURIInfo.RedirectURIQueryParams["user"].(map[string]interface{}); ok {
```

Comment:
The oAuthToken["user"] code below also needs updating

Reply:
Done